config/application.rb

```diff
@@ -31,5 +31,6 @@ class Application < Rails::Application
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
+    config.web_console.whitelisted_ips = '192.87.85.209'
```

Comment:
It gets rid of a bunch of warnings in the console. I was having trouble getting the site to load locally, and I thought this would fix my problem, but it didn't. I can get rid of it if you want, but I don't think it's causing any harm